### PR TITLE
Fix PDF generation bugs

### LINKED ISSUE #93: Update the metadata.yaml to accurately reflect PP instead of MB data
Part of PR #89

### LINKED ISSUE #96: Figure out missing book name
Part of PR #89

### LINKED ISSUE #188: Strip book, author, and translator yaml from PP chapter files
Follow-up to https://github.com/bountonw/translate/issues/96#issuecomment-2002904080.

### PR DESCRIPTION
**Pull request by mattleff**
Fixes #96. Fixes #93. Fixes #188. PDFs are [here](https://github.com/bountonw/translate/actions/runs/8321357948)

### CODE REVIEW

**Comment by bountonw** [comment]
Shouldnt the extra yaml be removed from all of the chapters and not just chapter 64?

**Comment by mattleff** [comment]
@bountonw Sure, I've removed it now: 14d7326.

**Comment by mattleff** [comment]
@bountonw Whoops, author and translator coming now...

File: lo/GC/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+lang: lo-LO
+author:
+  en: Ellen White
+  th: เอเลน ไวท์
+translator:
+  en: Brian Wilson
+  th: บุญต้น วิลสัน
+book:
+  title:
+    lo: TODO
```
Comment (mattleff) [inline lo/GC/metadata.yaml:11]: @bountonw I wasn't sure what to put here, so I left it TODO...

**Review by bountonw — COMMENTED** [review]
@mattleff Sometimes the order is 

```
th:
en:
```

Sometimes it is 
```
en:
th:
```

Shouldn't this be standardized?

**Comment by mattleff** [comment]
@bountonw Yes, we should probably standardize the order. I'm not finding any tools to lint that right off, so would it be alright to make an issue and come back to it?

**Comment by bountonw** [comment]
That would work.

**Comment by mattleff** [comment]
@bountonw Ok, #191 up. I think this is ready to merge otherwise.

**Review by bountonw — APPROVED** [review]

**Comment by bountonw** [comment]
@mattleff Approved. Refactoring the yaml of the translation files causes the clock to be reset on these files. They now all show the latest update as yesterday.

**Comment by mattleff** [comment]
> @mattleff Approved. Refactoring the yaml of the translation files causes the clock to be reset on these files. They now all show the latest update as yesterday.

Yeah, that's unfortunate. I'm not sure the best way to avoid that without adding a lot of complexity. Do you have any ideas?

**Pull request merged**